app/models/candidate.rb

```diff
@@ -6,9 +6,9 @@ class Candidate < ApplicationRecord
   before_validation :downcase_email
   validates :email_address, presence: true,
                             uniqueness: { case_sensitive: false },
-                            length: { maximum: 250 },
+                            length: { maximum: 256 },
```

Comment:
Will this need to be represented in the API docs too?

Reply:
I cannot find anything about this in the API docs, so I'm not sure. Do you have an idea? @fofr

Reply:
The max length in the API docs is 100 chars, and iirc this is a hard constraint from vendors

Reply:
Oh, then, I think we should change this to `100`, thanks @duncanjbrown

Reply:
Resolved in ff060ae